src/typography/src/Link.js

```diff
@@ -1,6 +1,6 @@
-import React, { memo, forwardRef } from 'react'
-import PropTypes from 'prop-types'
 import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React, { forwardRef, memo } from 'react'
```

Comment:
nit -- keep react imports at the top?

Reply:
heh, these were all automated by "organize imports"